content/docs/tasks/traffic-management/debugging-istio/index.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Debugging Istio
+description: Shows how to debug Pilot and Envoy.
+weight: 66
+keywords: [debug,proxy,status,config]
+---
+
+This task demonstrates how to use the [proxy-status](/docs/reference/commands/istioctl/#istioctl-proxy-status)
+and [proxy-config](/docs/reference/commands/istioctl/#istioctl-proxy-config) commands.
```

Comment:
Need add something to say why these commands are useful:

This task demonstrates XXX which let you diagnose Envoy configuration problems.

Or something like it.